BASS Audio Test/ObjectiveBASS.m

```diff
@@ -858,15 +858,14 @@ - (void)pause {
     });
 }
 
-- (void)ensureHasAudioSession {
-    if(AVAudioSession.sharedInstance.isOtherAudioPlaying) {
-        // needed to handle weird car bluetooth scenarios
-        [self setupAudioSession: NO];
-    }
+- (void)prepareAudioSession {
+    // needed to handle weird car bluetooth scenarios
+    BOOL shouldActivate = !(AVAudioSession.sharedInstance.secondaryAudioShouldBeSilencedHint);
```

Comment:
I made this change while trying to make Relisten not stop other app's audio on launch (which is going to require some different work with BASS). 
The `AVAudioSession` APIs say that `secondaryAudioShouldBeSilencedHint` is recommended over `isOtherAudioPlaying`. Let me know if this is the wrong change to make though, and I can back this out from the PR.